docs/multicluster/user-guide.md

```diff
@@ -592,43 +633,150 @@ In future releases, some additional tooling may become available to automate the
+feature mentioned in the previous section, and set the `clusterNetworkPolicy` field of
+the ResourceExport to the `acnp-drop-tenant-to-secured-mc-service` spec above. Such
+replication should only be performed by ClusterSet admins, who have clearance of creating
+ClusterNetworkPolicies in all clusters of a ClusterSet.
+
+### Ingress NetworkPolicy for Cross-cluster Traffic
+
+Antrea-native policies now support selecting ingress peers in the ClusterSet scope (since v1.10.0).
+Policy rules can be created to enforce security postures on ingress traffic from all member
+clusters in a ClusterSet:
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: drop-tenant-access-to-admin-namespace
+spec:
+  appliedTo:
+  - namespaceSelector:
+      matchLabels:
+        role: admin
+  priority: 1
+  tier: securityops
+  ingress:
+  - action: Deny
+    from:
+    # Select all Pods in role=tenant Namespaces in the ClusterSet
+    - scope: clusterSet
+      namespaceSelector:        
+        matchLabels:
+          role: tenant
 ```
 
-Normally, `podCIDRs` should be the value of `kube-controller-manager`'s
-`cluster-cidr` option. If it's left empty, the Pod-to-Pod connectivity feature
-will not be enabled. If you use `kubectl edit` to edit the ConfigMap, then you
-need to restart the `antrea-mc-controller` Pod to load the latest configuration.
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: AntreaNetworkPolicy
+metadata:
+  name: db-svc-allow-ingress-from-client-only
+  namespace: prod-us-west
+spec:
+  appliedTo:
+  - podSelector:
+      matchLabels:
+        app: db
+  priority: 1
+  tier: application 
+  ingress:
+  - action: Allow
+    from:
+    # Select all Pods in Namespace "prod-us-west" from all clusters in the ClusterSet (if the 
+    # Namespace exists in that cluster) whose labels match app=client
+    - scope: clusterSet
+      podSelector:        
+        matchLabels:
+          app: client
+  - action: Deny
+```
+
+As shown in the examples above, setting `scope` to `clusterSet` expands the scope of the
+`podSelector` or `namespaceSelector` of an ingress peer to the entire ClusterSet that the
+policy is created in. Similar to Egress NetworkPolicy, the scope of `appliedTo`
+is also restricted to the local cluster.
+
+To use the ingress cross-cluster NetworkPolicy feature, the `enableStretchedNetworkPolicy`
+option needs to be set to `true` in `antrea-mc-controller-config`, for each `antrea-mc-controller`
+running in the ClusterSet. Refer to the [previous section](#multi-cluster-pod-to-pod-connectivity)
+on how to change the ConfigMap:
+
+```yaml
+  controller_manager_config.yaml: |
+    apiVersion: multicluster.crd.antrea.io/v1alpha1
+    kind: MultiClusterConfig
+    ...
+    enableStretchedNetworkPolicy: true
+```
+
+Note that currently ingress stretched NetworkPolicy only works with the Antrea `encap`
+traffic mode, with tunnel type set to `Geneve`.
```

Comment:
Really? I assume all tunnel types should work (as they all support tunnel key)? At least VXLAN should work as it supports VNI too.

Reply:
@GraysonWu to confirm

Reply:
I didn't test it with other tunnel types. But based on my understanding and the [doc](https://www.openvswitch.org/support/dist-docs/ovs-fields.7.txt) I agree with Jianjun.
```
Many kinds of tunnels support a tunnel ID:
              •      VXLAN and Geneve have a 24-bit virtual network identifier
                     (VNI).
              •      LISP has a 24-bit instance ID.
              •      GRE has an optional 32-bit key.
              •      STT has a 64-bit key.
              •      ERSPAN has a 10-bit key (Session ID).
              •      GTPU has a 32-bit key (Tunnel Endpoint ID).
```
I could double-check it with a test. One more thing: ERSPAN whose tunnel ID is only 10 bits, which is not enough for LabelIdentity. Not sure if this is a valid use case. If yes, maybe we should mention StretchedNetworkPolicy is not supported when tunnel type is ERSPAN?

Reply:
We only support VXLAN, Geneve, GRE, STT as tunnel types, if multi-cluster uses same configuration. Perhaps just remove "with tunnel type set to `Geneve`"

Reply:
Done

Reply:
This PR is also required for this tunnel type update. https://github.com/antrea-io/antrea/pull/4516